src/applications/static-pages/i18Select/hooks.js

```diff
@@ -0,0 +1,42 @@
+export const onThisPageHook = (content, lang) => {
+  if (lang && lang !== 'en') {
+    const onThisPageEl = document?.getElementById('on-this-page');
+    onThisPageEl.innerText = content[lang].onThisPage;
+  }
+};
+
+export const setLangAttribute = lang => {
+  const contentDiv = document?.getElementById('content');
+  if (contentDiv) {
+    contentDiv.setAttribute('lang', lang);
+  }
+};
+
+// set lang code as `en` for all links on the current page
+// unless it follows the old IA lang patterns
+// without overwriting existing onclick events
+// reference: https://stackoverflow.com/questions/891989/javascript-adding-an-onclick-handler-without-overwriting-the-existing-one
+
+export const adaptLinksWithLangCode = setLangAttributeInReduxStore => {
+  const links = document.links;
+  for (let i = 0; i !== links.length; i++) {
```

Comment:
Would it be possible to use `forEach`?

Reply:
hello! 
forEach wasn't possible bc it is a NodeList and not an array, but I updated the loop to use `for...of` syntax... i agree that the old approach isn't as readable. thanks!